lib/Dashboard.js

```diff
@@ -0,0 +1,54 @@
+var rp = require('./request-promise'),
+  config = require('../config'),
+  _ = require('lodash'),
+  Q = require('q');
+
+var Module = require('./Module');
+
+function Dashboard (slug) {
+  this.options = {
+    json: true,
+    slug: slug
+  };
+}
+
+Dashboard.prototype.getConfig = function () {
+  var options = _.extend(this.options, {
+    url: config.stagecraft + 'public/dashboards?slug=' + this.options.slug
+  });
+
+  return rp(options);
+};
+
+Dashboard.prototype.resolve = function () {
```

Comment:
Could the naming of getConfig and resolve better indicate their relationship? 
getConfig gets a 'shallow' dashboard config without modules
resolve gets that plus modules
the names just seem unrelated, could make the API more intuitive?